test/live/live_test.go

```diff
@@ -0,0 +1,41 @@
+package live_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"attestation-service-examples.test"
+)
+
+const liveTestConfigTemplate = `
+acceptable_measurements = [
+{ platform = "{{.LIVE_TEST_PLATFORM }}", code = "{{.LIVE_TEST_CODE}}" },
+]
+auth_token = "{{.LIVE_TEST_AUTH_TOKEN}}"
+host = "{{.LIVE_TEST_HOST}}"
+`
```

Comment:
I like the fact that the non-live tests use each example project as-is. To perform live tests, though, we need to "override" the config. So, in the `live_test` package I supply a config template that we can populate using environment variables. It is then rendered and saved as a file in the testscript working directory.